pretixprint/app/src/main/java/eu/pretix/pretixprint/byteprotocols/BrotherRaster.kt

```diff
@@ -0,0 +1,193 @@
+package eu.pretix.pretixprint.byteprotocols
+
+import android.graphics.Bitmap
+import android.graphics.Matrix
+import eu.pretix.pretixprint.R
+import eu.pretix.pretixprint.ui.BrotherRasterSettingsFragment
+import eu.pretix.pretixprint.ui.SetupFragment
+import java8.util.concurrent.CompletableFuture
+import java.io.ByteArrayOutputStream
+import java.io.InputStream
+import java.io.OutputStream
+import kotlin.math.min
+
+// https://download.brother.com/welcome/docp100278/cv_ql800_eng_raster_100.pdf
+// https://download.brother.com/welcome/docp100366/cv_ql1100_eng_raster_100.pdf
+class BrotherRaster : StreamByteProtocol<Bitmap> {
+    override val identifier = "BrotherRaster"
+    override val nameResource = R.string.protocol_brother
+    override val defaultDPI = 300
+    override val demopage = "demopage_8in_3.25in.pdf"
+
+    // Label data list extracted from
+    // https://download.brother.com/welcome/docp100366/cv_ql1100_eng_raster_100.pdf
+    enum class Label(val id: Int, val width: Int, val height: Int, val continuous: Boolean, val printableWidth: Int, val printableHeight: Int, val twoColor: Boolean = false) {
+        c12mm(257, 12, 0, true, 106, 0),
```

Comment:
``width`` and ``printableWidth`` are apparently different units, I think we should at least add a comment in here so that a future reader of the code does not need to guess

Reply:
done :)